app/src/main/java/org/piwigo/bg/UploadService.java

```diff
@@ -208,8 +208,11 @@ public void onResponse(@NonNull Call<ImageUploadResponse> call, @NonNull Respons
                     }
                     onUploadStarted(imageUploadQueue);
                 } else {
+                    String add = "()";
```

Comment:
I think we should init it to "", such that in case we don't have additional info we also don't add empty parenthesis. Or we add response.raw().code() - which seems NonNull...

As this is also a long-lasting issue from the betas - I remember to have switched "up" and response.body().up_stat to call equals never on null, but I wonder when this happens and why. - I really like to understand the root-cause for this.